catalogue_pipeline/id_minter/src/test/scala/uk/ac/wellcome/platform/idminter/IdMinterFeatureTest.scala

```diff
@@ -88,7 +87,7 @@ class IdMinterFeatureTest
     val miroId = "1234"
     val sqsMessage = generateSqsMessage(miroId)
 
-    sqsClient.sendMessage(idMinterQueue, JsonUtil.toJson(sqsMessage).get)
+    sqsClient.sendMessage(idMinterQueue, toJson(sqsMessage).get)
```

Comment:
Should we modify the signature of `sendMessage()` to take a queue and an instance of SQSMessage, and have it do the JSON encoding?

(Again, could be a separate patch!)